modules/oe_theme_content_publication/src/Plugin/ExtraField/Display/PublicationDescription.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_content_publication\Plugin\ExtraField\Display;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\extra_field\Plugin\ExtraFieldDisplayFormattedBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Display EU contribution and its percentage of the total budget.
+ *
+ * @ExtraFieldDisplay(
+ *   id = "oe_theme_content_publication_description",
```

Comment:
In case you update the id of the extra field for the publication date, this one could be:
```suggestion
 *   id = "oe_publication_description",
```

Reply:
Best practice is to use module name in the plugin id. In this project plugins have different naming - with and without module name in id. So let's keep it as is until another team members will ask to change it.